Okta-456278: Update Sign in with password and email guide (#3750)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/index.md

<ApiLifecycle access="ie" />

> > **Note:** In proxy model architectures, where a server-side application using the Embedded SDK is used as a proxy between client applications and Okta servers, a request context for the client applications is required. Security enforcement is expected to be based on the client request context's IP address and user agent. However, since these values are currently being derived from the server application rather than the client, this enforcement is not available. As a result, network zones or behaviors that drive their conditions based on these request context values (geolocation, IP Address, or user agent) will not work until we can find a solution to the issue.

Reviewer: ```suggestion
> **Note:** In proxy model architectures, where a server-side application using the Embedded SDK is used as a proxy between client applications and Okta servers, a request context for the client applications is required. Security enforcement is expected to be based on the client request context's IP address and user agent. However, since these values are currently being derived from the server application rather than the client, this enforcement isn't available. As a result, network zones or behaviors that drive their conditions based on these request context values (geolocation, IP Address, or user agent) won't work until we can find a solution to the issue.
```

Author: fixed

---

## Overview

> With the Embedded SDK, your app can verify a user's identity using first a password and then the email authenticator. The email authenticator can complete its verification using [magic links](/docs/guides/email-magic-links-overview/aspnet/main/), a one-time password (OTP), or a combination of both.

Reviewer 2: ```suggestion
With the Embedded SDK, your app can verify a user's identity using a password and then the email authenticator. The email authenticator can complete its verification using [magic links](/docs/guides/email-magic-links-overview/aspnet/main/), a one-time password (OTP), or a combination of both.
```

Reviewer 2: The magic links link needs to be lang specific.

<div class="half">

> ![A representation of the required password and email factors](/img/oie-embedded-sdk/factor-password-email.png)

Reviewer 2: The image is sized to small.

Author: Are you sure?  Moving up to three-quarters width makes it look very large indeed.

Author: Updated

Reviewer 2: I agree the image looks somewhat too large now. Can include a size in the middle of those two sizes? If not then keep as is for now.
